lib/entity/domain-contacts.php

```diff
@@ -253,37 +255,56 @@ public function set_billing( Domain_Contact $billing ): void {
 	}
 
 	/**
-	 * Functions to implement the Iterator interface
+	 * Part of the iterator interface implementation
 	 *
 	 * @internal
+	 *
+	 * @return Domain_Contact|null
+	 * @throws \Automattic\Domain_Services\Exception\Entity\Invalid_Value_Exception
```

Comment:
We already import `Exception` above, so probably `Exception\Entity\Invalid_Value_Exception` is shorter and easier to scan/read